src/main.rs

```diff
@@ -247,7 +260,28 @@ impl App {
     fn next(&self, args: &ArgMatches) -> Result<()> {
         let next_handle = self
             .handle_from_args(args)
-            .or_else(|_| self.random_handle())?;
+            .or_else(|_| {
+                if args.get_flag("reverse") {
+                    let max_attemps = 100;
+                    for _i in 0..max_attemps {
+                        let handle = self.random_handle().unwrap();
+                        let clash = self.read_clash(&handle).unwrap();
```

Comment:
When a function returns a `Result` and you're calling another function that also returns a result, you can use Rust's `?` syntax instead of `.unwrap()`. That way the error is propagated and the caller can handle it if they wish (whereas `.unwrap()` would just panic right here).